src/puppetlabs/puppetdb/query/paging.clj

```diff
@@ -216,17 +216,8 @@
     ((some-fn nil? (every-pred coll? empty?)) order_by)
     (not count?))))
 
-(defn rename-first
-  "rename the first element of a vector according to kmap"
-  [kmap pair]
-  (update-in pair [0] #(% kmap)))
-
 (defn dealias-order-by
   [{:keys [projections] :as query-rec} paging-options]
-  (let [alias-map (reduce-kv (fn [acc k v]
-                               (assoc acc
-                                 (keyword k)
-                                 (-> v :field h/extract-sql)))
-                             {} projections)]
-    (update-in paging-options [:order_by]
-               #(map (partial rename-first alias-map) %))))
+  (let [alias-map (->> (kitchensink/mapvals (comp h/extract-sql :field) projections)
```

Comment:
I'd agree that if there we're a functional change it would need to be in its own commit, maybe even a note about it in the commit message, but a separate commit I'm not sure about